cardano-cli/cardano-cli.cabal

```diff
@@ -230,7 +230,7 @@ test-suite cardano-cli-golden
   build-depends:        aeson             >= 1.5.6.0
                       , base16-bytestring
                       , bytestring
-                      , cardano-api
+                      , cardano-api ^>= 8.1.0.1
```

Comment:
`cardano-cli-test` , `cardano-cli-golden`, `depend on cardano-cli. The version bounds in `cardano-cli` should be enough to correctly resolve dependency version and the version bounds here seem redundant.